public/index.html

```diff
@@ -117,20 +117,20 @@
 package main 
 
 import (
-  "os"
+	"os"
 )
 
 templ greeting(name string, imgURL string) {
-  <div style="background-color: lightblue; padding: 20px;">
-    <h1>Hello, { name }</h1>
-    <img src={ imgURL } alt="Welcome image" style="width: 200px; height: auto;" />
-    <p>Welcome to our site. We're glad to have you here!</p>
-  </div>
+	<div style="background-color: lightblue; padding: 20px;">
```

Comment:
Doesn't indenting like this cause errors if you load the page and just try to run? it had caused it to misbehave for me in the past (I'll give it a little test on my local I was just curious if you'd noticed the same behaviour and maybe your IDE/formatter had just done this)

Reply:
Not for me, @joerdav and I noticed before that the example when we implemented the format button got formatted. So now just went to the example and pasted the formatted code and it seems to respect it. Curious though if it does break for you

Reply:
It was github's diff being weird, once i pull it down there's no indentation at all on that example code, can ignore.